src/components/TextField/TextField.tsx

```diff
@@ -175,22 +181,29 @@ const TextField: VibeComponent<TextFieldProps, unknown> & {
 
     return (
       <div
-        className={classNames("input-component", wrapperClassName, {
-          "input-component--disabled": disabled
+        className={cx(styles.textField, "input-component", wrapperClassName, {
+          [styles.disabled]: disabled,
+          ["input-component--disabled"]: disabled
         })}
         role={role}
         aria-busy={loading}
       >
-        <div className="input-component__label--wrapper">
+        <div className={cx(styles.labelWrapper, "input-component__label--wrapper")}>
           <FieldLabel labelText={title} icon={labelIconName} iconLabel={iconsNames.layout} labelFor={id} />
           <div
-            className={classNames("input-component__input-wrapper", SIZE_MAPPER[getActualSize(size)], validationClass)}
+            className={cx(
+              styles.inputWrapper,
+              "input-component__input-wrapper",
+              SIZE_MAPPER[getActualSize(size)],
```

Comment:
Should also add css-modules classes here. Check out `TextFieldConstants.ts` file
Should look smth like this 
![image](https://user-images.githubusercontent.com/104433616/225280614-1d960c29-4ae0-4c6c-8858-1690d1276ba5.png)